query-coordinator/src/test/scala/com/socrata/querycoordinator/rollups/TestRollupQueryRewriter.scala

```diff
@@ -0,0 +1,172 @@
+package com.socrata.querycoordinator.rollups
+
+import com.rojoma.json.v3.util.{AutomaticJsonCodecBuilder, JsonUtil}
+import com.socrata.querycoordinator.QueryRewriter.{ColumnId, RollupName}
+import com.socrata.querycoordinator._
+import com.socrata.soql._
+import com.socrata.soql.collection.OrderedMap
+import com.socrata.soql.environment.{ColumnName, DatasetContext}
+import com.socrata.soql.functions.{SoQLFunctionInfo, SoQLTypeInfo}
+import com.socrata.soql.parsing.{AbstractParser, Parser}
+import com.socrata.soql.typed.CoreExpr
+import com.socrata.soql.types.{SoQLType, SoQLValue}
+
+import scala.io.Source
+
+
+case class TestCase(query: String, rewrites: Map[String, String])
+object TestCase {
+  implicit val jCodec = AutomaticJsonCodecBuilder[TestCase]
+}
+
+case class TestConfig(schemas: Map[String, Map[String, (SoQLType, String)]], rollups: Map[String, String], tests: Seq[TestCase])
+object TestConfig {
+  implicit val jCodec = AutomaticJsonCodecBuilder[TestConfig]
+}
+
+
+class TestRollupQueryRewriter extends TestBase {
+
+  //
+  // System Under Test
+  //
+  val analyzer = new SoQLAnalyzer(SoQLTypeInfo, SoQLFunctionInfo)
+  val rewriter = new QueryRewriterWithJoinEnabled(analyzer)
+
+  //
+  //  Helper methods
+  //
+  private def analyzeQuery(q: String,
+                   context: AnalysisContext[SoQLType, SoQLValue],
+                   columnMapping: Map[QualifiedColumnName, String]):
+  BinaryTree[SoQLAnalysis[ColumnId, SoQLType]] = {
+
+    val parserParams = AbstractParser.Parameters(allowJoins = true)
```

Comment:
This is identical to the other parserParams. Can we pull it into a shared scope and resuse it in both places?